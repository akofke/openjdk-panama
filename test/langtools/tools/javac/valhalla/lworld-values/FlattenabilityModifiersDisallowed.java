/*
 * @test /nodynamiccopyright/
 * @summary Verify that flattenability modifiers are not exposed by default.
 * @compile/fail/ref=FlattenabilityModifiersDisallowed.out -XDrawDiagnostics -XDdev FlattenabilityModifiersDisallowed.java
 * @compile -XDrawDiagnostics -XDdev -XDallowFlattenabilityModifiers FlattenabilityModifiersDisallowed.java
 */

class FlattenabilityModifiersDisallowed {
    __Flattenable int x;
    __NotFlattened int y;
}
