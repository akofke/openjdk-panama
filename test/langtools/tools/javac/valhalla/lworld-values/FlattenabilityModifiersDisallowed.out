FlattenabilityModifiersDisallowed.java:9:5: compiler.err.flattenability.modifiers.disallowed
FlattenabilityModifiersDisallowed.java:10:5: compiler.err.flattenability.modifiers.disallowed
2 errors
